test/converters/xcresult3/action_test_plan_summary.go

```diff
@@ -39,9 +39,10 @@ type Name struct {
 }
 
 // tests returns ActionTestSummaryGroup mapped by the container TestableSummary name.
-func (s ActionTestPlanRunSummaries) tests() map[string][]ActionTestSummaryGroup {
+func (s ActionTestPlanRunSummaries) tests() ([]string, map[string][]ActionTestSummaryGroup) {
 	summaryGroupsByName := map[string][]ActionTestSummaryGroup{}
 
+	testSuiteOrdering := []string{}
```

Comment:
NOTE: based on this: https://github.com/bitrise-steplib/steps-deploy-to-bitrise-io/pull/152/files#diff-f1f03b66d8394133fe7409796d0c4dd9628abd604720db68b01bd3dbe8be15e1R28

`testSuiteOrdering` could be `testSuiteOrder` or the other way around